orbit/pkg/table/pmset/pmset_darwin.go

```diff
@@ -41,7 +41,7 @@ func Generate(ctx context.Context, queryContext table.QueryContext) ([]map[strin
 		}
 	}
 
-	output, err := exec.CommandContext(ctx, "/usr/bin/pmset", "-g", getting).CombinedOutput()
+	output, err := exec.CommandContext(ctx, "/usr/bin/pmset", "-g", "custom", getting).CombinedOutput()
```

Comment:
Isn't this achieved by:
```
SELECT * FROM pmset WHERE getting = 'custom';
```
?

We'll have to double check the macOS CIS policies that use this table continue to work.

I just checked and we are lucky it seems the output of:
```sh
/usr/bin/pmset -g custom
```
is the same as:
```
/usr/bin/pmset -g custom custom
```

Reply:
So that's pretty unusual for a table isn't it? That the column actually adds an argument to the command to get data? Also, why doesn't "everything" work if that's how it works?

Reply:
At the time, we implemented the bare minimum to get CIS policies for macOS up and running.

> So that's pretty unusual for a table isn't it? That the column actually adds an argument to the command to get data?

Yeah, tables built as wrapper for macOS commands...

Reply:
Well, personally I wish they all worked like this. :)